src/main/java/com/bzabek/ticketbookingapp/entities/Reservation.java

```diff
@@ -0,0 +1,146 @@
+package com.bzabek.ticketbookingapp.entities;
+
+import com.bzabek.ticketbookingapp.model.TicketType;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import static com.bzabek.ticketbookingapp.Configuration.RESERVATION_EXPIRE_TIME_MIN;
+
+
+@Entity
+public class Reservation {
+    @Id
+    @GeneratedValue
+    private UUID id;
+
+    private boolean paid;
+    private boolean active;
+    private LocalDateTime expirationTime;
+    private String viewerName;
+    private String viewerSurname;
+    @OneToOne
+    private SeatReserved seatReserved;
+    @Enumerated(EnumType.STRING)
+    private TicketType ticketType;
+
+    public TicketType getTicketType() {
```

Comment:
czy te wszystkie gettery i settery są potrzebne? jeśli naprawdę tak, to polecam poczytać o lomboku